---
title: Patch notes of The Division 2
meta:
    description: All patch notes from the release on forward of The Division 2
---

This page contains all the patch notes that were published since the release of the game. Newest patch notes are always at the top.

## Patch from 14. May 2019: Operation Dark Hours (Title Update 3)

*New Feature: Operation Dark Hours*

Operation Dark Hours is our first 8-player raid coming to Tom Clancy’s The Division 2. Teamwork, collaboration, and adaptation will be necessary to defeat the Black Tusk. More information and exact opening times will be revealed soon. 

*New Feature: Classified Assignments: The Fall of DC*

Classified Assignments are exclusive missions for owners of the Year 1 Pass. Discover and complete these missions to hear stories behind the fall of DC as well as unlocking exclusive Backpack Trophies. Title Update 3 includes the first two Classified Assignments.

*New Apparel Event: Dark Hours*

Starting together with the raid release, our second Apparel event will introduce new outfits, masks, emotes and weapon skins! 

*ACHIEVEMENTS & TROPHIES*

* Added Operation Dark Hours Achievement / Trophy.

*COMMENDATIONS*

* Added new Commendations for Operation Dark Hours.
* Removed the following Commendations
** 28 Days Distinction - Record 672 hours (28 days) of time in-game.
** Distinguished Service Ribbon – For 30 consecutive days, on each day play for at least 1h and complete 1 mission.
** Supreme Response Distinction - For 7 consecutive days, on each day complete 4 main missions and rescue 5 civilians.
** Year One Merit - Play on 14 days per month, on a total of 12 months.
** Division Service Merit - For a total of 100 days, on each day play for at least 1hour and deliver 20 resources to control points.
** *Developer comment:* We removed these Commendations as we felt that they were not delivering a great experience for players. Often times you could miss out on your daily login or lose your streak with no error of your own. We’re adding different Commendations that still let you show off your dedication towards the game, without the login requirements. For those players that have already earned the above-mentioned Commendations they will remain visible as completed.
** Added a new vanity patch reward for all players as compensation for any lost time due to the above changes.
* Added five new Commendations
** Full Deck Distinction – Acquire all Snitch Cards.
** Distinguished Service Distinction – Complete 30 hours of service.
** Supreme Response Distinction – Complete all main missions and Strongholds on Hard difficulty (or above).
** Control Point Takeover Merit – Take over 50 Heroic Control Points.
** Invasion Service Merit – Complete 10 Invaded Strongholds.
* Changed Peacock Award to specifically use the 'Jumping Jacks' emote.
* Changed Mission Discovery Merit Commendations to track when a player collects loot from loot boxes, not opening the loot boxes. This will ensure that all players that are in a group can earn this Commendation when playing the respective missions.

*CRAFTING*

* Increased the base cap for Receiver Components and Protective Fabric crafting materials by 200, meaning that their caps now start at 350 and end at 600 with all material capacity perks.
* Inaya al-Khaliq in the Base of Operation is now a Crafting Vendor, she can be found next to the Crafting Bench in the White House.
** Added Specialized and Superior Skill Mod Blueprints to her stock.
** Added an extra weekly Blueprint to her stock.
** Moved crafting Blueprints from the Base of Operations vendor to her stock.
** Note: While Inaya will have a variety of Blueprints available, there are still other sources in the game like Settlements and Control Points.
* All exotic items upgraded through crafting will get the maximum Gear Score of the current crafting bench tier.

*GEAR*

* Bonus Armor granted on blue attributes gear significantly increased.
* Armor percent can now be found as Defense Attribute rolls.
* Health percent can now be found as Defense Attribute rolls.
* Gear sets
** Hard Wired
*** Now grants Skill Damage instead of Shock Duration.
** Ongoing Directive
*** Now grants special ammunition into any weapon, including the weapon currently equipped.
*** Now grants special ammunition on any kill, not just weapon kills.
** True Patriot
*** Decreased debuff cycle from 4 seconds to 2 seconds.
*** Increased damage dealt to nearby targets when Full Flag is active to 50%.

*ICONS*

* The Confidential Gear Set intel item now has a project icon to better reflect its use.

*LOOT*
* Loot containers will now reset at a preset time of day. 
** Control Point reward container will now also reset with this daily timer.
* Improved quality and quantity of Drone Helicopter loot.
* Slightly reduced the quality of items from Clan Caches.
* Cabinet loot containers now have a chance to contain Dark Zone keys.
* Increased ammunition drop chance from enemy NPCs to limit situations where players would run out of ammunition during prolonged fights on higher difficulty activities.
* Dark Zone contaminated loot does not drop below player's average Gear Score and will guarantee Gear Score 500 when the player has reached 500 Gear Score.
* Heroic difficulty Mission bosses, Stronghold bosses, Bounty bosses and Heroic Control Points reward containers will guarantee Gear Score 500 when the player has reached 500 Gear Score.
* Loot now skews more towards the top end of its allowed Gear Score, resulting in fewer items below a player’s average Gear Score, and items that do drop below it will be closer to that average more often.
* Increased the rewards for daily Hard and Challenge mission Projects and they will guarantee Gear Score 500 when the player has reached 500 Gear Score.

*MISSIONS*

* New feature: Post Mission Activity Summary. After completing missions, players may opt to look at an activity summary showcasing their performance in that activity and allowing an expanded look at a variety of detailed statistics.
* Bank Headquarters
** Reduced difficulty for solo players.
* Roosevelt Island
** Reduced difficulty for solo players.

*MODS*

* Hive users can now find +healing on the Experimental Blend mods.
* Hive users can now find +radius on the Network Firewall mods.
* New mods: Auxiliary Skill Battery
** Auxiliary Skill Batteries fit in Skill mod slots and grant Skill Power.

*NPC*

* Updated RC Blade to fix various issues.
* Elite Medic’s Pulse Grenade has been changed.
** No longer applies disrupt.
** Reduced Impact radius.
** The grenade impact radius should now be easier to see.
* Decreased damage of Veteran, Elite and Named NPCs.
* Greatly decreased melee damage of all Tank NPCs.
* Moderately decreased melee damage of all non-Tank NPCs.
* Decreased Thrower NPC’s accuracy with grenades. They are no longer professional baseball pitchers or cricket fielders.
* Slightly decreased damage of Elite/Named War Hound.
* Civilian Control Point Officers now wear pants.
* Civilian Control Point Officers now wear protective body armor. Safety first.
* Civilians with fully upgraded Settlements now wear combat harnesses and have slightly more health.
* Upgrading a Settlement now slightly increased Civilians’ damage.
* NPCs will now be slightly less defensive while in cover and being aimed at by a player.
* Decreased likelihood of NPCs using ladders in combat.
* Decreased likelihood of NPCs using special navigation out of combat.
* Civilian NPCs will now stay further away from players in cover, lessening the chance of them taking a player’s preferred cover location. We get it, you’re very protective of your favorite cover spot.
* Civilian Hostages are now more protective of their lives.
* Fixed issue with many explosives causing extreme amounts of threat, confusing NPCs.
* Fixed various sliding issues with NPCs following recovery from a Status Effect.
* Enemy NPCs should no longer be able to stagger players with a melee attack if the player has a Ballistic Shield equipped.
* Drone Helicopter
** Increased health.
** Increased damage.
** Improved movement.
* Decreased Outcast Suicide vest explosion radius.
* Lowered NPC damage and health above normal difficulty.
* Increased variety of NPCs at higher difficulty levels.

*OPEN WORLD*

* Implemented various UI and audio improvements to Living World activities.
* Control Points
** Control Point difficulty is now more clearly communicated, using regular difficulty names in addition to Alert Level.
** Approaching enemy patrols and convoys are now signaled more clearly during Control Point Takeover.
** The Civilian Officer will now revive downed players during Control Point takeover and judge you silently.
** Players can now select the civilian takeover icon on the map as a respawn option during Control Point takeover. The option is disabled after the enemy leader joins the fight.
** Players can now fast travel to civilian convoys and takeover squads from the map.
* Black Tusk now send War Hound Convoys carrying gear and weapons to Tidal Basin when the mission is available.

*RECALIBRATION*

* Recalibration has been changed to allow for the majority of stats to be moved as they are, from one item to another, while making it less likely to reach the cap of the stat moved. To achieve this goal, we’re expressing the recalibrated power as a separate number called Recalibration Score, next to the Gear Score. As such, recalibration will no longer increase the gear score of the item. All existing recalibrated items will have their recalibrated additional gear score converted to the new format.
* Adjusted cost and ability to be recalibrated for
** Terminate
** Knee Cap
** Calculated

*REWARDS*

* XP from Bounties has been increased.
* All Settlement Weekly Projects now award blueprints.
* New daily Project: SHD Requisition. 
** Requires crafting ingredients and rewards high quality gear and blueprints.
* Increased the quality of Weekly Project equipment rewards.

*SKILLS*

* Assault Drone
** PvE: The Drone will now more actively look for a new target once it loses its current target.
* Defender Drone
** Will now drain more quickly once a shot has been deflected.
* Reinforcer Chem Launcher
** Reinforcer gas clouds do not stack anymore.
* Reviver Hive
** Fixed further occurrences of players not being revived.
** Revive time slightly increased.
* Riot Foam Chem Launcher
** Increased base duration of Riot Foam when used against a NPC.
* Scanner Pulse
** Reduced cooldown from 120 seconds to 90 seconds.

*GEAR TALENTS*

* Berserk
** Increased required max armor depleted from 10% to 20%.
** Now requires 7 Offense Attributes.
* Blacksmith
** Increased Armor repair from 25% to 50%.
** Can now occur every 15 seconds, was 10 seconds.
** Now requires 7 Defense Attributes.
* Bloodlust
** PvE
*** Increased granted Weapon Damage from 25% to 35%.
*** Decreased duration from 10 seconds to 5 seconds.
** PvP
*** Decreased granted Weapon Damage from 25% to 20%.
*** Decreased duration from 10 seconds to 5 seconds.
*** Now requires 9 Offense Attributes.
* Bloodsucker
** PvE
*** Depleting an enemy’s armor adds a stack of +25% bonus armor for 10s. Max stack is 6. (was 20%, 20s, 5)
** PvP
*** Depleting an enemy’s armor adds a stack of 20% bonus armor for 5s. Max stack is 5.
*** Now requires 11 Offense Attributes and an Assault Rifle, SMG or LMG equipped.
* Critical
** Reduced Critical Hit Damage bonus from 8% to 5%.
* Clutch
** Increased Health gained per Critical Hit from 15% to 20%.
** Reduced Armor gained per Critical Hit from 2% to 1%.
** Now requires 4 or less Defense Attributes.
* Entrench
** Increased Armor repair for Headshots from cover from 5% to 10%
* Hard Hitting
** Reduced Damage to Elites from 15% to 10%.
* Knee Cap
** Increased chance to apply bleed when Shooting an Enemy in the legs from 10% to 15%.
* Patience
** After being in cover for 5s, armor repairs by 5% every 1s. (was 3s).
** Now requires 6 Defense Attributes
* Safeguard
** Reduced Duration from 20 to 5 seconds.
** Can only occur once every 20 seconds.
** This was already the case but is now reflected in the tooltip.
** Now requires 4 or less Offense Attributes.
* Spotter
** Increased Weapon Damage to Pulsed enemies from 10% to 20%.
** Now requires 5 Utility Attributes.
* Surgical
** Reduced Critical Hit Chance bonus from 8% to 5%.
* Unbreakable
** PvE
*** Increased amount of Max Armor repaired when armor is depleted from 25% to 70%.
*** Increased timeframe for Armor Kits not being consumed when effect triggers from 5 seconds to 7 seconds.
** PvP
*** Increased Max Armor repaired when armor is depleted from 15% to 25%.
*** Increased timeframe for Armor Kits not being consumed when effect triggers from 5 seconds to 7 seconds.
*** Now requires 11 Defense Attributes and no other Talents affecting Armor Kits.
* Unstoppable Force
** PvE
*** Killing an enemy now grants 2% Weapon Damage for every 25.000 Max armor.
** PvP
*** Killing an enemy now grants 1% Weapon Damage for every 25.000 Max armor.
*** Now requires 7 or more Defense Attributes.
*** Developer comment: Due to the increase in available armor on gear made we decided to also increase the Max armor requirements for Unstoppable Force.

*WEAPON TALENTS*

* Frenzy
** PvE
*** For every 10 bullets in a magazine capacity, gain +2% rate of fire and +2% weapon damage for 5s when reloading from empty.
** PvP
*** For every 10 bullets in a magazine capacity, gain +1.5% rate of fire and +1.5% weapon damage for 5s when reloading from empty.
* Killer
** Now grants 30% critical damage for 5s instead of 50% critical hit chance.
** Developer comment: We think critical damage feels more appropriate as you already need a relatively high critical hit chance to activate the talent regularly.
* Measured
** Now grants 20% rate of fire and -15% weapon damage in the top half of the magazine.
** Now grants -20% rate of fire and +30% weapon damage in the bottom half of the magazine
** Developer comment: Measured should now be a net damage bonus and the increased rate of fire should better support other talents and effects activating.
* Preservation
** PvE:
*** Killing an enemy repairs 10% armor over 5s. Headshots improves the repair to 20%. (was 5%, 3s, 10%)
** PvP
*** Killing an enemy repairs 5% armor over 3s. Headshots improves the repair to 15%.
*** Now requires 7 Defense Attributes.
* Reformation
** Increased Headshot kill skill repair and healing to 50% from 25%.
** Reduced duration from 25 seconds to 20 seconds.
** Now requires 9 Utility Attributes.
* Spike
** Increased skill damage from 25% to 35%.
** Increased duration from 10 seconds to 20 seconds.
** Now requires 5 Utility Attributes.
* Strained
** Increased missing Armor requirement from 5% to 10%.
* Unhinged
** Reduced Damage Bonus from 25% to 20%.
** Reduced Handling negative from 35% to 25%.

*VENDORS*

* Vendors will now verify the players' highest achieved World Tier and offer items of that Gear Score, instead of offering items of the highest World Tier available in the group.

*WEAPONS*

* New feature: Exotic Mod Bonuses. All exotic mod slots now have unique bonuses.
* SOCOM MK20 SSR 
** Now has a skin slot from Specialized quality and up.
* Nemesis
** Mod slots
*** Optic: +35% Headshot Damage
*** Magazine: +10% Reload Speed
*** Muzzle: +15% Critical Hit Damage
*** Rail: +15% Damage to Elites
* Chatterbox
** Damage increased.
** Mod slots
*** Optic: +5% Critical Hit Chance
*** Magazine: +10% Reload Speed
*** Muzzle: +10% Critical Hit Chance
*** Rail: +15% Handling
* Sweet Dreams / Lullaby
** Damage increased.
** Mod slots
*** Optic: +15% Accuracy
*** Magazine: +15% Reload Speed
*** Rail: +25% Optimal Range
* Merciless / Ruthless
** Damage increased.
** Mod slots
*** Optics: +10% Accuracy
*** Magazine: +15% Reload Speed
*** Muzzle: +20% Stability
*** Rail: +20% Handling
* Pestilence
** Damage increased.
** Mod slots
*** Magazine: +10% Rate of Fire
*** Muzzle: +20% Stability
*** Rail: +15% Handling
* Liberty
** Damage increased.
** Mod slots:
*** Optic: +10% Critical Hit Chance
*** Magazine: + 15% Damage to Elite
*** Muzzle: + 15% Stability
*** Rail: + 15% Rate of Fire
* TAC-50
** Now has 2 zoom states.

*CONFLICT*

* Armor boost
** Boost provided reduced by 50%.
* Armor value on nameplates
** Improved visuals to better reflect how much armor an enemy has.
* Balance
* All players are now scaled to Max World Tier Gear Score.
* Primary stats are set to high-end values.
* Normalized talent overrides are applied.
* Budget distribution of un-normalized items is preserved.
* No additional bonus is applied.
* Unlocked mods and talents from un-normalized builds are active.
* Items above Gear Score 500 will be scaled to 500.
* No bonus stats will be applied to these items.
* Recalibration bonuses are not applied.
* PvP specific overrides
** Weapon and Skill global damage modifiers.
** Targeted weapon and skill damage modifiers.
*** Assault Rifles now have a 1.25 PvP damage scaler applied.
** Skill behavior overrides.
* Damage boost
** Boost provided reduced by 50%.
* The Conflict NPC was moved to a more prominent spot, next to the helipad outside the Base of Operations.
* Respawn
** Respawning players are now much more likely to respawn with their teammates.
* Skills
** Skill cooldowns no longer reset when a player is killed.

*PVP*

* Damage modifier reduced to increase time to kill.
* Skill modifier increased to let Skills have more impact in PvP.

*DARK ZONE*

* Contaminated loot
** Bosses now guarantee a contaminated loot drop.
** Higher challenge landmarks can drop multiple contaminated items.
** Basic, Veteran and Elite NPCs have their drop rate lowered, but now have a very high chance to drop contaminated loot.
* DZ bag size
** Default bag size increased to 6.
** Tier 1 perk Bag Space now increases bag size to 8.
** Tier 4 perk Bounceback now reduces Dark Zone XP lost on death.
** Tier 10 perk increases bag size to 10.
* DZ Brackets
** Reduced bracketing to:
** Level 1 – 30.
** Players in World Tiers.
* NPC AI
** Reduced damage output.
** Increased Time to Kill.
* Landmarks
** Added a short cooldown to landmarks.
* Rogue
** Removed cooldown.
** There is still a 5 second toggle cooldown.
* Thieves Den
** Now allows players to switch weapons.
* Occupied Dark Zone 
** Damage modifiers
** Added a separate PvP damage modifier.
* XP
** Reduced DZXP loss to 35% of the XP in your current Dark Zone rank when being killed.

*GENERAL*

* Text Chat improvements
** Players can toggle displaying personal and group loot drops.
** Text chat will now lose focus after sending a message.
** Group tab will be populated with all raid members when queueing for a raid.
** Improved feedback when attempting invalid actions in text chat.

*SETTINGS*

* Added a Neutral Lighting setting to all platforms.

*PC*

* Settings
** Added "Anti-aliasing" graphics setting with new Ultra alternative for sharper image when in motion.
* UI
** Character sheet
*** Will now properly update and display Accuracy, Stability, Recoil, Weapon Swap and Reload Speed from +Weapon Handling bonuses.
** Social menu
*** When downed, agents now have the possibility to open the Social menu via hotkey.
*** Players will now be able to matchmake during Heroic difficulty missions.
*** Players are now immune to Vote Kicks during boss encounters and for a short time after the end of boss encounters and missions. Going AFK removes this immunity.

*BUG FIXES*

* Fixed an issue where NPCs in the Bounty screen had broken animations. Put your hands down, please.
* Fixed an issue that could prevent players from completing the Side Mission Medical Camp Attack.
* Fixed an issue where armor values on AI in the Dark Zone scaled incorrectly.
* Fixed an issue where the Sniper Turret would aim at targets out of line of sight in PvP.
* Fixed an issue where Scanner Pulse and Remote Pulse were not properly affected by Skill Duration.
* Fixed various issues where Skill bonuses were not applied properly.
* Fixed an issue where bounties rewarded from Projects in Settlements were rewarding the wrong amount of XP and Credits.
* Fixed an issue where Clan Cashes would scale incorrectly when opened in another player’s lower level session.
* Fixed a bug where NPCs from a Resource Convoy could spawn out of thin air on top of the player.
* Fixed a bug where the Control Point supply room would remain locked when capturing a Control Point too quickly.
* Fixed several bugs with matchmaking that prevented correct matching of players based on their region, language, gear score and other criteria.
* Fixed an issue with ‘A Friend in Need’ Achievement/Trophy which would cause it to not unlock in certain conditions.
* Fixed an issue where a player would be unable to complete the ‘Big Game Hunter’ Achievement/Trophy.
* Fixed an issue on the Ubisoft Club Weekly Challenges screen on PC that resulted in a low frame rate when on that screen.
* Fixed an issue causing Korean language glyph compositions not displaying properly in the in-game chat.

## Patch from 2. May 2019

* Fixed an issue where players could not interact with Henry Hayes.

## Patch from 30. April 2019

* We've released a small patch for the PC client that should improve the long loading times some players have been experiencing.

## Patch from 25. April 2019

* Fixed an issue affecting slow loading times on PC.

## Patch from 18. April 2019

* Fixed an issue where players were able to receive unlimited Snitch heroic bounties.
* Fixed another issue where players were stuck after being revived while having a DoT state active.
* Fixed other issues relating to players being stuck after revival.
* Lowered NPC damage and health above normal difficulty.
* Increased variety of NPCs spawned at higher difficulty levels.
* Difficulty has been significantly decreased in Hard, moderately reduced in Challenge, and barely reduced in Heroic.
* Fixed an issue related to Elite NPC aggressiveness in Heroic / Alert Level 4.
* Reduced the damage output of Named bosses.

## Patch from 16. April 2019

* Fixed an issue where enemy NPCs could get stuck in a T-pose when being killed out of view.
* Fixed another issue with the Revive Hive not working properly. We are still working on instances where the skill sometimes will not resurrect players.
* Fixed an issue where the Gila Guard brand gear could not be donated to the Campus Daily Project "Brand Request: Gila Guard".
* Fixed an issue where the Crafting Bench upgrade blueprint would not be granted to players when moving from one World Tier to another.
* Fixed an issue where the two piece bonus of the True Patriot gear set would not trigger correctly.

## Patch from 12. April 2019

* Fixed an issue that could cause the Demolitionist's tactical link buff to stack infinitely

## Patch from 11. April 2019

* Fixed not being able to fast travel to the Castle settlement.
* Fixed several cases of abnormal Bounty boss skill-use behaviour.
* Fixed an issue causing the Nemesis crafting materials to be unobtainable.
* Fixed Delta-03 error occurring at the end of Conflict matches.
* Fixed an issue that would cause the pathfinding line to behave abnormally when in close proximity to your destination.
* Fixed the "donating" sound playing repeatedly after having donated and abandoned a daily mission.
* Fixed an issue related to cache countdown timers.
* Fixed an issue where Specialization ammunition drop rate was lower than intended.

## Patch from 5. April 2019, Invasion: Battle for D.C.

* New content
** Invasion: Battle for D.C. is now live. You can find a content overview in [this article](https://tomclancy-thedivision.ubisoft.com/game/en-us/news-updates/346408/tidal-basin-is-coming).

* Weapons, Mods and Gear
** Overhauled the Weapon mod system, see the table at the end of the patch notes for detailed information. (Or our [weapon mods list](/weapon-mods.html))
** Fixed a wrong long-range damage value for M4 (super 90) and P416. Previously, they could sometimes increase damage over longer ranges instead of the other way around.
** Rifles:
*** Damage on the MK17 rifle reduced by 14%, also corrected this weapon not having a damage falloff over distance.
*** LW M4 damage increased by 5%, RPM increased from 240 to 360.
*** LVOA-C RPM increased from 240 to 380.
** Marksman Rifles
*** Model 700 damage reduced by 13%.
** Shotguns
*** AA12 damage increased by 16%.
** LMGs
*** MG5 damage increased by 12% and fixed reversed falloff, causing the weapon to do more damage over a longer range.
** Updated initial accuracy on the Sig Sniper MCmillan T1. It should now be much more reliable to shoot as soon as it’s shouldered and the transition to scoped view is over.
** Reduced the amount of Critical Damage and Headshot Damage that can roll on Gear, including the amount on existing Gear. The actual values will differ depending on the level and quality of the gear it occurs on. This is partially in preparation for another world tier, where we felt these values were already at the limit of what was healthy in WT4.
** Complete overhaul of skill mods to give more achievable requirements and more reasonable bonuses.
*** A closer equivalence between required skill power and the bonuses granted.
*** Changed the amount of skill power on gear to not increase exponentially with level, but instead more linearly. This results in most skill power values on gear being lower, but in accordance with the lowered requirements.
*** Tweaked the ranges of bonuses to be within acceptable bounds where they had extreme values before due to scaling errors. Most notably radius and ammo/charges values.
*** Players will see both the bonuses and the requirements of almost all skill mods change as a result of this overhaul.
*** In end game, both Superior and High-End skill mods exist to cover the different needs of different builds. They can overlap in power, but High-End mods have a higher maximum roll potential.
*** We are looking at ways to provide lower value/quality mods in end game world tiers in the future to provide a solution to the fact that on very low skill power builds, there would be room for those mods, and currently we are aware that means you have to save mods from your leveling experience.

* Player Skills
** Fixed an issue which caused the Bombardier Drone to get destroyed while climbing up/down a ladder or rope.
** Improved feedback of Bombardier Drone crashing into obstacles during the bombing phase.
** Fixed an issue with Firefly payload sometimes failing to activate when reaching its target.
** Fixed an issue where Revive hive would not self-deploy if the agent was busy performing a different action such as using different skill when it triggered.
** Revamped the Sniper Turret version of the turret skill.
*** It now will automatically target the hostile nearest to the players’ center-of-screen (the same method it uses to pick other turret targets) and will fire at that target when the button is pressed, without a lock-on order needed.
*** If a player aims at a hostile, the sniper turret will attempt to match their aim, allowing precision shots using the sniper turret.
*** An issue where the sniper turret wasn’t correctly getting bonus damage when hitting headshots has been fixed.
** Revamped the Chem launcher skill platform.
*** Pressing the skill button will now equip the chem launcher similar to swapping to a weapon. The player can aim and shoot with normal weapon controls, as well as run with the skill and blindfire it.
*** Quick deploy at the agents own position is still available for the repair version of the chem launcher, using the same controls as before.
*** Unequipping the Chem launcher can be done using the skill button, the B (XBOX) / Circle (PS4) button or just using the weapon swap controls.

* Loot & Rewards
** Control Point reward containers are giving the correct number of items for the corresponding Control Point's defense level. Defense level 4 now awards 5 items instead of 3.
** Reduced DZ XP gain from Landmarks in Dark Zones.
** Reduced the quality and power for items dropped in Propaganda Broadcast and one drop of Settlement Blockade activities to be more in line with other open world enemies and activities.
** Updated XP rewards for low-level player playing in World Tiers to scale with player level.
** Supply Drop activity loot container items now scale with every consecutive container you open. This includes lowering the reward of the first crate, but results in similar rewards for the activity as a whole.
** Added 5 Specialized Apparel Cache Keys, awarded with levels 2 – 4.
*** As a thank you for playing, all players that played before April 4 get 5 Specialized Apparel Keys, regardless of their level.
*** Apparel Keys will be coming for those players later next week.
** Faction Key loot drops from Bounties are now guaranteed and are displayed in the loot preview.
** Adjusted bounty credit rewards to use scaling values in World Tiers instead of flat values so if a lower level player is playing with a higher-level player, they will receive an amount of credits more appropriate for their level.
** Changed Skill Unlock rewards to be non-invaded only so that players who co-op with a higher-level friend to play invaded before playing on story won't receive the rewards.
** Added Apparel cache key fragments to the PvP reward caches.
** Adjusted the rewards for the street execution activity to give slightly lower quality/source level gear so the rewards are more in line with the difficulty of the activity.
** Reduced the quality and source level charts used for item drops from the living world activities settlement blockade and Propaganda broadcast to be more in line with other open world drops and the difficulty of the activities.
** Updated Collectibles set XP reward to scale with player level.
** Fixed an issue which caused the Drone Helicopter to not use the correct loot table.

* Crafting 
** Increased Material cap for Standard and Specialized crafting materials to 400 (with all perks).
** Increased Material cap for High-End crafting materials to 50.
** Materials required for crafting weapon mods does not scale as aggressively and has much lower values in endgame.
** Removed a small increase on materials required for crafting weapon mods after upgrading the crafting bench quality.
** Changed the upgrade blueprints for exotic weapons Lullaby and Ruthless to not show up on the vendor until you are level 30.
** Increased the power for the wandering vendor so her inventory has a narrower Gear Score range closer to the top of the World Tier Gear Score Range.
** Set all exotic upgrade blueprints to award you the upgraded weapon on the max gear score of the current tier.
** Removed a tiny increase on materials required for crafting weapon mods after upgrading the crafting bench quality.
** Exotic weapons are now pre-equipped with lore text mods. These mods do not have stats and cannot be equipped elsewhere.

* Talents
** Blind Justice – Modified the Buff that increases damage to your next shot now expires at end of combat and when out of combat
** First Blood - Updated Normalized MMR Talent First Blood Text to correctly explain that the first shot after reloading from empty is the enhanced bullet.

* NPCs
** Decreased damage of all NPC Automatic Shotguns
** Decreased damage of all NPC SMGs
** Hey agent! Staff in the Base of Operation will now shout at the player less frequently than before.

* Main Missions Difficulty
** World Bank – Invaded – Server Room
*** ​Increased the time which the Black Tusks needs to complete the download in the Server Room from 200 seconds to 250 seconds.
*** Reduced the difficulty in the Server Room by setting the Enemy group composition to light, light, medium - Instead of medium, heavy, medium.
** DCD Headquarters
*** Mission level has been changed from 17 to 18.
** Roosevelt Island – Invaded – Quarantine Docks
*** Reduced the difficulty of the first wave of enemies encountered in the last beat of the mission by lowering the amount of Elite and Veteran enemies.
*** Enemies in this part of the mission can only use mounted weapons if there are 2 players or more.
**** This should make it less punishing for solo-players attempting this mission.

* Open World
** Removed death of civilian squad as a failure condition for Control Point takeover.
*** The Officer will now remain in downed state until players complete the takeover or wipe.
** Black Tusk presence has been increased in invaded Open World zones.
** New Ambusher NPCs now roam the streets of DC.

* Social
** Added a “Friends & Clan” option for Call for Backup that only shows calls from friends and clan members.
** Added a “Do Not Disturb” group privacy option that blocks all incoming invites.

* PC
** Added a Field of View slider

* PlayStation 4
** Fixed an issue that could cause players to be stuck in the Specialization tutorial

* Bug Fixes
** Fixed an issue that could cause undesired appearance changes to agents after login in or using the barber.
*** Players that have been permanently affected by an appearance change will unfortunately not be changed back with this fix. We want to apologize for any inconvenienced caused by this and we’re looking into options to allow players to fully customize their agent in the future, but we do not have a timeline for this feature.
** Fixed an issue where the Cluster Seeker Mine could split up more often than intended.
** Fixed an issue where the Defender Drone could have unlimited uptime.
** Fixed an issue which prevented players from finishing the Unite cypher riddle.
*** We're confident this is not bugged anymore.
** Fixed an issue that would cause the Stronghold activation animation to always play when entering the Base of Operations.
** Fixed several instances of Audio logs not playing automatically when picked up.
** Fixed an issue that could cause players being unable to finish the Side Mission Worksite Community.
** Fixed an issue where players were unable to update their crafting bench after reaching a new World Tier.
** Fixed an issue with NPCs sometimes not reacting properly to cooked grenades
** Fixed an issue with Hunters not always appropriately despawning if a player flees a combat space
** Fixed rare issue where NPCs could get stuck when attempting to melee a player in cover
** Fixed issue with the Explosive Seeker Mine occasionally failing to path to its target
** Fixed bug where players could fast travel to non-friendly Control Points
** Fixed bug that made encounters with friendly NPCs requesting resources too rare
** Fixed bug where underground enemies would come to the surface during the day
** Fixed an issue where players were able to obtain gear with a higher gear score than intended
*** Any owned High-End items will be lowered to Gear Score 500.
*** Any owned Superior items will be lowered to Gear Score 490.
*** Clan caches no longer award items higher than the maximum loot Gear Score.
*** Recalibration still allows Gear Score to increase slightly past the maximum loot Gear Score.
** Fixed a bunch of weird stairs and invisible walls.

## Patch from 28. March 2019

* Performance
** Various improvements to reduce frame drops.

* Clans
** Changed the Clan Member UI colour to not conflict with Elite enemies.
** Fixed an issue where players Clan XP contributions could stop registering.

* VOIP
** Changed Push-to-Talk to be the default VOIP setting.

* UI
** Changed Apparel Store purchase confirmations to a button hold action.
** Reduced the UI flickering effect present in the menu.
** Fixed a visual bug that would incorrectly display a grey duplicate backpack reward when opening apparel caches.
** Fixed the menu distortion animation playing twice under certain circumstances.

* Skills
** Fixed abnormal skill behaviour when equipping a skill after having switched to a grenade.
** Fixed instances of skills going on cool down when deploying a shield.
** Fixed an issue where multiple Cluster Seeker Mines could be deployed in short succession.

* Mods
** Fixed an issue where donating a weapon with mods attached would remove the mods from the players inventory.

* Gameplay
** Fixed daily and weekly missions not resetting for offline players.
** Improvements made to Co-op health scaling.
** Fixed an issue where armour/gear scaling would not take World Tiers into account.
** Fixed skill damage incorrectly being scaled twice for low level players joining a high level group.
** Fixed players potentially becoming stuck on the “Meet The Division Coordinator” objective after having played in Co-op.
** Fixed players potentially becoming stuck on the “Meet with Odessa Sawyer” objective.

* Open World
** Fixed an issue that would allow players to upgrade neutral control points.

* Side Missions
** Fixed an issue that would cause low level players to gain unintended amounts of XP by completing side missions with a high level group member.

* PC
** Fixed an issue where some Logitech headphone users experienced missing in-game audio.

* UI
** Fixed “Navigate Categories” and “Mark as Junk” being bound to the same button in some occurrences.

* Xbox One
** Fixed an issue where the HDR setting slider would not move.

## Patch from 22. March 2019

* Fixed an issue where players were not being granted Clan XP
* Fixed an issue where daily and weekly resets would not apply to players who were offline during the time of reset
* Fixed an issue where players could lose guiding missions and access to guiding mission NPCs, making them unable to further progress
* Improved survivability of low level players against enemy NPCs when playing co-op with a player of higher level

## Patch from 16. March 2019

* Bug fixes
** Implemented a fix that should reduce occurrences of client crashes when using skills
** Fixed an issue where players could be stuck in the wrong World Tier if they completed Strongholds without meeting the required Gear Score while in a group
** Fixed an issue where players wouldn't be able to talk to agent Kelso at the Castle Settlement
** Fixed an issue where players could obtain multiple Ivory Keys from the same source
** Fixed an issue where map progress would not be saved when completing Open World activities and Side missions in another players’ instance.
** Fixed an exploit that allowed players to loot Supply Drops multiple times
** Fixed a bug where players could buy blueprints repeatedly at the vendors by logging out and back in
** Fixed an issue where combining the Dialed in talent with the Emet Mask could lead to greatly increased Accuracy and Stability

## Patch from 14. March 2019

* Weapons
** We removed a work in progress weapon that was not properly named and balanced.
** Crossbows now deal explosive damage to multiple objects as intended
** Exotic weapons equipped with silencers will now produce the sound they’re intended to make.
** NPCs engaged in combat will now react to grenades as intended.

* Missions
** The ropes will now deploy properly to allow for extraction at the end of the Beekeeper Jeff side mission.
** Players should no longer be able to unlock Castle prematurely. This issue would block progression in some instances.
** In Jefferson Plaza, the NPC scene can no longer be interrupted during a boss fight.
** Fixed a loot exploit that involved repeatedly killing bosses in some Invaded missions.

* PC specific
** PC players playing together while connected to the same network were meeting a number of issues when interacting with each other. This has now been fixed.
** On PC, FreeSync 2 HDR can now properly be selected in game regardless of your PC’s configuration.

* Other
** Stopped "SHD CPU V.2" and "Cyclone Magazine" skill mods from dropping as loot. Player's who already have the items can use them once they unlock the corresponding specialization perk.
** Players will no longer get stuck on black screens after watching introduction cinematics.
** We have updated XP rewards for all end game activities, so that they will scale to the player’s level, instead of being flat values.
** Several corrupted audio issues have been fixed.

## Title update 1.5

* Title Update 1.5 mainly addresses occurrences of audio missing or cutting out experienced by some during Open Beta.
